src/gentables/CMakeLists.txt

```diff
@@ -2,13 +2,13 @@ cmake_minimum_required(VERSION 3.5)
 
 # remove $CC from the current environment and by that force cmake to look for a (working) C compiler,
 # which hopefully will be the host compiler
-unset(ENV{CC})
+set(ENV{CC} "$ENV{CC_FOR_BUILD}")
```

Comment:
Naming it `HOST_CC` or `FLUID_HOST_CC` makes it clearer. Likewise for the other vars below.

Reply:
Sorry, but the names are already taken. I haven't chosen them myself and I would have chosen them differently. But the *_FOR_BUILD suffices are established since literally decades, at least in the Autotools ecosystem.

https://www.sourceware.org/autobook/autobook/autobook_143.html#Supporting-Cross-Compiler-in-Make
https://www.gnu.org/software/autoconf-archive/ax_prog_cc_for_build.html
https://manpages.debian.org/bookworm/autotools-dev/config.guess.1.en.html
https://stackoverflow.com/a/40954603

Reply:
I see. I'm no autotools guy, so thanks for the pointer.